tests/.gitignore

```diff
@@ -1 +0,0 @@
-mbed-os
```

Comment:
what is the issue in ignoring mbed-os ?

Reply:
The mbed-os symbolic link in `TESTS/LinkLoss/device` was being ignored. However, `bootstrap.sh` now adds the symbolic links so this is no longer an issue.